src/main/java/org/team1540/robot2025/subsystems/elevator/ElevatorIOTalonFX.java

```diff
@@ -138,7 +137,12 @@ public void setSetpoint(double setpointMeters) {
     public void setBrakeMode(boolean brakeMode) {
         leader.setNeutralMode(brakeMode ? NeutralModeValue.Brake : NeutralModeValue.Coast);
         follower.setNeutralMode(brakeMode ? NeutralModeValue.Brake : NeutralModeValue.Coast);
-        // follower.setControl(followerControl);
+    }
+
+    @Override
+    public void resetPosition(double positionMeters) {
+        leader.setPosition(positionMeters);
+        follower.setPosition(positionMeters);
```

Comment:
does setting the follower position do anything?

Reply:
It makes their positions synced in the logs so we can check if the follower diverges from the leader